scripts/timelapse-colorizer-data/convert_nucmorph_data.py

```diff
@@ -168,13 +172,22 @@ def make_dataset(output_dir="./data/", dataset="baby_bear", do_frames=True, scal
     reduced_dataset[INITIAL_INDEX_COLUMN] = reduced_dataset.index.values
     grouped_frames = reduced_dataset.groupby(TIMES_COLUMN)
 
+    # Get the units and human-readable label for each feature; we include this as
+    # metadata inside the dataset manifest.
+    features = ["NUC_shape_volume_lcc", "NUC_position_depth"]
```

Comment:
I feel like this string list is now defined twice in the file and should be DRY factored up?